### DISCUSSION TITLE
Error on web access

### DISCUSSION THREAD

**Discussion by jsamaniegog** [glpi-project/glpi-agent#754] [Q&A]
### Bug reporting acknowledgment

Yes, I read it

### Professional support

None

### Describe the bug

```
# **systemctl status glpi-agent**
● glpi-agent.service - GLPI agent
   Loaded: loaded (/usr/lib/systemd/system/glpi-agent.service; enabled; vendor preset: disabled)
   Active: active (running) since vie 2024-09-06 11:29:58 CEST; 17s ago
     Docs: man:glpi-agent
  Process: 1622 ExecReload=/bin/kill -HUP $MAINPID (code=exited, status=0/SUCCESS)
 Main PID: 1708 (glpi-agent: wai)
   CGroup: /system.slice/glpi-agent.service
           └─1708 glpi-agent: waiting

sep 06 11:29:58 hcuve05561 systemd[1]: Started GLPI agent.
sep 06 11:29:59 hcuve05561 glpi-agent[1708]: GLPI Agent starting
sep 06 11:29:59 hcuve05561 glpi-agent[1708]: [http server] HTTPD ToolBox Server plugin loaded
sep 06 11:29:59 hcuve05561 glpi-agent[1708]: [http server] HTTPD service started on port 62354
sep 06 11:30:00 hcuve05561 glpi-agent[1708]: target server0: next run: Sat Sep  7 05:45:04 2024 - https://glpi.my.domain/glpi/
sep 06 11:30:09 hcuve05561 glpi-agent[1708]: [http server] unknown path: *%D8U$%9D%EF%9A%FD%C0%AB%9C%0F%F2_%D9@%AE%81%8C%CCbr%3E%BA%AFj%BD%C8V%EF%80;%00
sep 06 11:30:09 hcuve05561 glpi-agent[1708]: [http server] unknown path: %01%00%07%1C%03%03
sep 06 11:30:09 hcuve05561 glpi-agent[1708]: [http server] unknown path: %01%00%07%1C%03%03%F5%C0b%8E$i%86c%17%B1%07%5E%8D2%1DcrO%0BW%D4%A6R%18V3%18%CB(%04iS
sep 06 11:30:09 hcuve05561 glpi-agent[1708]: [http server] unknown path: ,C%D6%90og%08%A9%BFB%8E%DB%j%10%95

```

### To reproduce

?

### Expected behavior

Access URL `https://my.agent.server:62354/`

### Operating system

Linux

### GLPI Agent version

v1.10

### GLPI version

10.0.15

### GLPIInventory plugin or other plugin version

Not applicable

### Additional context

This has suddenly stopped working. I have tried with version 1.9 and 1.10.
The browser displays: ERR_SSL_PROTOCOL_ERROR

**Comment by g-bougard** [comment]
Hi @jsamaniegog 

the "unknown path" errors are due to a web client on your network trying to make http access on agent listening port. Such unknown path looking may be related to SSL communication, but glpi-agent is not configured to support SSL request by default.

You should probably enable debug=2 in configuration so glpi-agent will also report remote client ip. With the remote client ip, maybe this will help you to discover what is making that requests and decide is you can ignore them or not.

But looking at your "Additional context" infos, may I suppose the web client is just you trying to access your agent using `https://` in your URL ? If this is the case, just try `http://` in your url. And if you want to have SSL access to your client, you may be interested by configuring GLPI-Agent SSL server plugin: https://glpi-agent.readthedocs.io/en/latest/plugins/ssl-server-plugin.html ... but if you also need GLPI to request inventory run, this may break this feature.
